src/vector/text/vtextshaper.cpp

```diff
@@ -0,0 +1,244 @@
+#include "vtextshaper.h"
+
+
+std::string fontKey(FontProperty prop) {
+    std::string key = std::string(prop.family);
```

Comment:
check for null property  .. otherwise string will assert .. 
replace std::string key = std::string(prop.family); with std::string key{prop.family}; to reduce 1 copy of string.